### PR TITLE
Added French translations for v0.8.0 [#620]

### PR DESCRIPTION
**Pull request by BRUCELLA2**
Added French translations for v0.8.0

## Changes Need To Be Discussed First

Before opening a pull request, please be sure to announce your changes in [the
developer mailing list](https://www.freelists.org/list/comixed-dev) (and subscribe if you
haven't already). This will help speed up getting your pull request reviewed and merged.

**NOTE:** You can erase any parts of this template that are not applicable to your PR,
including this top section.

## Status
READY

## Does this PR contain migrations?
NO

## Before You Submit Your PR:
- [x] Have you announced your PR on the comixed-dev mailing list?
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: comixed-web/src/assets/i18n/fr/admin.json [hunk 1]
```diff
@@ -1,28 +1,29 @@
...
-      "confirmation-title": "Clear Web Audit Log",
-      "effect-failure": "Failed to clear the web audit logs.",
-      "effect-success": "Web audit logs cleared."
+      "confirmation-message": "Êtes-vous sûr de vouloir vider le journal d'audit ? Cette action est irreversible.",
+      "confirmation-title": "Vider le journal d'audit Web",
+      "effect-failure": "Échec lors de la suppression du journal d'audit Web.",
+      "effect-success": "Journal d'audit Web vidé."
     },
     "label": {
       "actions": "Actions",
```
Comment (mcpierce) [inline comixed-web/src/assets/i18n/fr/admin.json:10]: Please translate this entry.

File: comixed-web/src/assets/i18n/fr/app.json [hunk 2]
```diff
@@ -1,91 +1,91 @@
...
+      "build-host": "Le code a été build sur la machine nommée {name}",
+      "build-time": "Le code a été build le {time}.",
+      "build-version": "Le version du code est : {version}.",
+      "commit-id": "Le hash du dernier commit de la branche source est {id}.",
+      "commit-message": "Message du commit",
+      "commit-time": "Le commit a été créé le {time}.",
+      "commit-user": "Le commit a été créé par {name} <{email}>",
+      "dirty": "La branche {dirty, select, true{contient} other{ne contient pas}} de modification non comité.",
+      "jdbc-url": "L'url de la base de données utilisée est {url}",
       "remote-origin-url": "This branch is tracking the remote branch {url}"
```
Comment (mcpierce) [inline comixed-web/src/assets/i18n/fr/app.json]: Please translate this entry.

**Review by mcpierce — CHANGES_REQUESTED** [review]
Very good! Thanks for getting this so quickly.

Just need to add two missed entries, and remove the second line on the commit message and this is ready to merge.

File: comixed-web/src/assets/i18n/fr/admin.json (hunk 1 above)
Comment (BRUCELLA2) [reply comixed-web/src/assets/i18n/fr/admin.json:10]: Actions is translated as Actions it seems to me unless I don't understand its meaning correctly.

File: comixed-web/src/assets/i18n/fr/app.json (hunk 2 above)
Comment (BRUCELLA2) [reply comixed-web/src/assets/i18n/fr/app.json]: ok

File: comixed-web/src/assets/i18n/fr/admin.json (hunk 1 above)
Comment (mcpierce) [reply comixed-web/src/assets/i18n/fr/admin.json:10]: No, it's more likely due to my weaker french.

**Review by mcpierce — APPROVED** [review]

**Pull request merged**